pkg/installation_test.go

```diff
@@ -40,24 +41,50 @@ func TestCreateInstallationResources(t *testing.T) {
 		},
 	}
 
-	t.Run("when subscription is available then assign", func(t *testing.T) {
+	t.Run("when the *Installation resources are not present then it creates them", func(t *testing.T) {
```

Comment:
minor question: why do you have `*Installation` here?

Reply:
it's a short version for `CheInstallation` and `TektonInstallation` resources

Reply:
ah got it. I was currently dealing with other things and I thought it was a typo. My apologies :/